### PR TITLE
add php & ruby to verification docs

### PR DESCRIPTION
**Pull request by svix-frank**
Pending composer push

### CODE REVIEW

File: src/theme/CodeTabs.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export default function CodeTabs({ children }) {
             { label: 'JavaScript', value: 'js', },
             { label: 'Python', value: 'py', },
             { label: 'Go', value: 'go', },
+            ...additionalTabs,
```
Comment (svix-frank) [inline src/theme/CodeTabs.js:14]: This is temporary until we get the ruby & php libraries to parity

Comment (tasn) [reply src/theme/CodeTabs.js:14]: Yeah, sounds good!

**Review by tasn — APPROVED** [review]
Looks good to me.. Let me know once I can merge.

**Comment by tasn** [comment]
Merged all the rest of the PRs, so this is good to go, right?

**Comment by svix-frank** [comment]
> Merged all the rest of the PRs, so this is good to go, right?

Just need to tag a new release for packagist to pick it up and then we can merge this!

**Comment by svix-frank** [comment]
@tasn this is ready to be merged

**Pull request merged**